# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pivot-root.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Just some missing pivot_root.S inclusions ...

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.28/arm/dyn_syscalls.S	2004-12-13 11:21:31.000000000 +0100
+++ dietlibc-0.28-fixed/arm/dyn_syscalls.S	2005-02-03 21:26:56.431882216 +0100
@@ -107,6 +107,7 @@
 #include "../syscalls.s/pause.S"
 #include "../syscalls.s/personality.S"
 #include "../syscalls.s/pipe.S"
+#include "../syscalls.s/pivot_root.S"
 #include "../syscalls.s/poll.S"
 #include "../syscalls.s/query_module.S"
 #include "../syscalls.s/init_module.S"
--- dietlibc-0.28/i386/dyn_syscalls.S	2005-01-28 17:04:54.000000000 +0100
+++ dietlibc-0.28-fixed/i386/dyn_syscalls.S	2005-02-03 21:25:58.067754912 +0100
@@ -60,6 +60,7 @@
 #include "../syscalls.s/mprotect.S"
 #include "../syscalls.s/nanosleep.S"
 #include "../syscalls.s/pipe.S"
+#include "../syscalls.s/pivot_root.S"
 #include "../syscalls.s/poll.S"
 #include "../syscalls.s/readv.S"
 #include "../syscalls.s/stat.S"
